jetty-osgi/pom.xml

```diff
@@ -90,7 +90,7 @@
       </dependency>
       <dependency>
         <groupId>org.eclipse.jetty</groupId>
-        <artifactId>jetty-websocket</artifactId>
+        <artifactId>jetty-websocket</artifactId> <!-- TODO: is this even right? -->
```

Comment:
@janbartel while debugging I saw this, any idea what this is for? I don't know what this is referring to, I don't think there is any `jetty-websocket` artifactId in jetty 10 so I thought this should be an errror.